Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -270,28 +270,18 @@ public async Task EndToEnd_NoAPI()
         privateNuGetAssets.Delete(true);
     }
 
-    // These two are commented because the Github Actions runners don't let us easily configure the Docker Buildx config -
-    // we need to configure it to allow emulation of other platforms on amd64 hosts before these two will run.
-    // They do run locally, however.
-
-    //[InlineData("linux-arm", false, "/app", "linux/arm/v7")] // packaging framework-dependent because emulating arm on x64 Docker host doesn't work
-    //[InlineData("linux-arm64", false, "/app", "linux/arm64/v8")] // packaging framework-dependent because emulating arm64 on x64 Docker host doesn't work
-
-    // this one should be skipped in all cases because we don't ship linux-x86 runtime packs, so we can't execute the 'apphost' version of the app
-    //[InlineData("linux-x86", false, "/app", "linux/386")] // packaging framework-dependent because missing runtime packs for x86 linux.
-
-    // This one should be skipped because containers can't be configured to run on Linux hosts :(
-    //[InlineData("win-x64", true, "C:\\app", "windows/amd64")]
-
-    // As a result, we only have one actual data-driven test
-    [InlineData("linux-x64", true, "/app", "linux/amd64")]
+    [DockerSupportsArchInlineData("linux/arm/v7", "linux-arm", "/app")]
```

Comment:
are those being run at CI/locally at the moment?

Reply:
I've been running them on my local Windows machine, where I have buildx agents set up to support all of the non-skipped architectures in this PR. When we move to the SDK, it would be good to have a similar setup, so we can get these under CI :)  We can't have them run under github actions CI because the linux instances we run on do not support architecture emulation.